provider/terraform.rb

```diff
@@ -43,7 +43,7 @@ def tf_type(property)
     # "Namespace" - prefix with product and resource - a property with
     # information from the "object" variable
     def namespace_property_from_object(property, object)
-      "#{object.__product.prefix[1..-1].camelize}#{object.name}#{property.name.camelize}"
+      "#{object.__product.prefix[1..-1].camelize(:lower)}#{object.name}#{property.name.camelize}"
```

Comment:
will this work for apis that have names that are two words long, like resource manager?

Reply:
Nope! #1055 filed for general cleanup in this area. I could plumb `resource_name` through right now, but don't want to & would rather investigate this as a followup.